docs/dev/news/parsers.feat

```diff
@@ -0,0 +1 @@
+Parser combinators now available from the standard library
```

Comment:
I think you've earned a few more words on this than just that. :) Mention some of the cool features at least too!